Cargo.toml.orig

```diff
@@ -0,0 +1,162 @@
+[package]
+name = "surreal"
+publish = false
+edition = "2021"
+version = "1.0.2"
+license-file = "LICENSE"
+authors = ["Tobie Morgan Hitchcock <tobie@surrealdb.com>"]
+
+[features]
+# Public features
+default = ["storage-mem", "storage-rocksdb", "scripting", "http"]
+storage-mem = ["surrealdb/kv-mem"]
+storage-rocksdb = ["surrealdb/kv-rocksdb"]
+storage-speedb = ["surrealdb/kv-speedb"]
+storage-tikv = ["surrealdb/kv-tikv"]
+storage-fdb = ["surrealdb/kv-fdb-7_1"]
+scripting = ["surrealdb/scripting"]
+http = ["surrealdb/http"]
+http-compression = []
+ml = ["surrealdb/ml", "surrealml-core"]
+experimental_parser = ["surrealdb/experimental_parser"]
+
+[workspace]
+members = ["lib", "lib/examples/actix", "lib/examples/axum"]
+
+[profile.release]
+lto = true
+strip = true
+opt-level = 3
+panic = 'abort'
+codegen-units = 1
+
+[profile.bench]
+strip = false
+
+[dependencies]
+argon2 = "0.5.2"
+axum = { version = "0.6.20", features = ["tracing", "ws", "headers"] }
+axum-client-ip = "0.5.0"
+axum-extra = { version = "0.7.7", features = ["query", "typed-routing"] }
+axum-server = { version = "0.5.1", features = ["tls-rustls"] }
+base64 = "0.21.5"
+bytes = "1.5.0"
+<<<<<<< HEAD
+clap = { version = "4.4.11", features = [
+    "env",
+    "derive",
+    "wrap_help",
+    "unicode",
+] }
+=======
+ciborium = "0.2.1"
+clap = { version = "4.4.11", features = ["env", "derive", "wrap_help", "unicode"] }
+>>>>>>> upstream/main
+futures = "0.3.29"
+futures-util = "0.3.29"
+glob = "0.3.1"
+http = "0.2.11"
+http-body = "0.4.5"
+hyper = "0.14.27"
+ipnet = "2.9.0"
+ndarray = { version = "0.15.6", optional = true }
+once_cell = "1.18.0"
+opentelemetry = { version = "0.19", features = ["rt-tokio"] }
+opentelemetry-otlp = { version = "0.12.0", features = ["metrics"] }
+pin-project-lite = "0.2.13"
+rand = "0.8.5"
+<<<<<<< HEAD
```

Comment:
Just noticed that this file is from a merge conflict. The entire file can be deleted.

Reply:
Yes, I already noticed, I am still actively merging in the changes on main and I accidentally included .orig files in my commit.